akka-parsing/src/main/scala/akka/parboiled2/Parser.scala

```diff
@@ -114,15 +114,18 @@ abstract class Parser(
   private var _cursor: Int = _
 
   // the value stack instance we operate on
-  private var _valueStack: ValueStack = _
+  private var _valueStack: ValueStack = new ValueStack(initialValueStackSize, maxValueStackSize)
```

Comment:
So we're deviating from the 'upstream' implementation - seems fine, we can find the differences in git anyway

Reply:
Yeah, not sure it's still the same anyway.